tests/test_helper/lxd.bash

```diff
@@ -0,0 +1,40 @@
+LXC_TEST_IMAGE=jammy
+DEPENDENCIES="jq"
+
+function start_containers() {
+    LXC_COUNT=$1
+    LXC_BASE_NAME="microovn-test"
+    ALL_CONTAINERS=""
+    for (( i = 0; i < LXC_COUNT; i++ )); do
+        CONTAINER_NAME="$LXC_BASE_NAME-$i"
+        echo "# Launching $CONTAINER_NAME" >&3
+        lxc launch -q ubuntu:$LXC_TEST_IMAGE $CONTAINER_NAME
+        lxc_exec $CONTAINER_NAME "apt update && apt install -y $DEPENDENCIES"
+        ALL_CONTAINERS+="$CONTAINER_NAME "
+    done
+
+    export ALL_CONTAINERS
+}
+
+function cleanup_containers() {
+    IFS=" "
+    for container in $ALL_CONTAINERS ; do
+        echo "# Cleaning up $container" >&3
+        lxc delete --force $container
+    done
+}
+
+function lxc_exec() {
```

Comment:
To support a possible future of multiple backends, should we omit `lxd` in the name here and just do something like `instance_exec`? I guess a shorter name would be better though.